config/sync/config_ignore.settings.yml

```diff
@@ -0,0 +1,6 @@
+ignored_config_entities:
+  0: 'xmlsitemap.xmlsitemap.*'
+  2: 'webform.webform.*'
```

Comment:
We aren't going to be using webform right?

Reply:
I hope we don't open that up again but it is in our stack and a common request. It was the example default text so I included it.